modules/default/weather/weather.js

```diff
@@ -43,7 +43,8 @@ Module.register("weather", {
 		onlyTemp: false,
 		showPrecipitationAmount: false,
 		colored: false,
-		showFeelsLike: true
+		showFeelsLike: true,
+		absoluteDates: false
```

Comment:
if this is a property for the forecast, shouldn't this inserted in `weatherforecast.js` instead?

Reply:
I thought the current and forecast modules were deprecated in favor of the standalone weather module.

Reply:
ups, sorry for the noise, was irritated when looking in the tests (theoretically I should know this ...)

Reply:
😅👍🏻